tests/unit/amo/components/TestCategory.js

```diff
@@ -1,14 +1,16 @@
-import { shallow } from 'enzyme';
+import { shallow, mount } from 'enzyme';
 import React from 'react';
+import { Provider } from 'react-redux';
 
 import { CategoryBase, mapStateToProps } from 'amo/components/Category';
 import { SearchBase } from 'amo/components/Search';
 import createStore from 'amo/store';
 import { categoriesFetch } from 'core/actions/categories';
 import { searchStart } from 'core/actions/search';
 import { ADDON_TYPE_THEME } from 'core/constants';
+import I18nProvider from 'core/i18n/Provider';
 import { getFakeI18nInst } from 'tests/unit/helpers';
-
+import NotFound from 'amo/components/ErrorPage/NotFound';
```

Comment:
I don't think we have import sorting turned on. I think there's some linting based on groupings of internal and external imports etc but otherwise no other sorting. See also: https://github.com/airbnb/javascript/blob/c43fc749c4213e4574a4b4a3d0fe8784d5cd5b5e/packages/eslint-config-airbnb-base/rules/es6.js#L151